content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md

```diff
@@ -102,68 +97,77 @@ Go to **Apps**.
 
-   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudAddPostgresBackupVolume.png" alt="Add Host Path for Postgres Backup Volume" id="Add Host Path for Postgres Backup Volume" >}}
+   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudEnableHostPath.png" alt="Add Nextcloud Storage Host Path" id="Add Nextcloud Storage Host Path" >}}
 
-6. Select **Enable cronjobs for nextcloud** on the **CronJob configuration** screen.
+    You can set the ACL permissions here by selecting **Enable ACL** but it not necessary. You can also change dataset permissions from the **Datasets** screen using the **Edit** button on the **Permissions** widget for the Nextcloud **Data** dataset.
 
-   {{< expand "Nextcloud Cron Jobs" "v" >}}
-   NextCloud cron jobs only run while the app is running. If you stop the app, the cron job(s) do not run until you start the app again.
+   b. Select **Host Path (Path that already exsits on the system)** in **Type**, then browse to and select the **userdata** dataset to populate the **Host Path** for the **Nextcloud User Data Storage** fields.
+     
+   c. Scroll down to the **Nextcloud Postgres Data Storage** option.
+      Select **Host Path (Path that already exsits on the system)** in **Type**, then browse to and select the **pgpdata** dataset to populate the **Host Path**.
+    
+   d. Scroll down to **Nextcloud Postgres Backup Storage**, select **Host Path**, and then enter or browse to the path for the **pgbbackup** dataset.
+      When complete, the four datasets for Nextcloud are configured.
 
-   For more information on formatting and using cron jobs, see [Managing Cron Jobs]({{< relref "ManageCronJobsSCALE.md" >}}).
-   {{< /expand >}}
+    {{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageConfigPostgressHostPaths.png" alt="Add Nextcloud Storage Volumes" id="Add Nextcloud Storage Volumes" >}}
+
+6. Accept the remaining setting defaults.
 
-7. Accept the remaining setting defaults on the **Scaling/Upgrade Policy** and **Advanced DNS Settings** screens.
+7. Scroll up to review the configuration settings and fix any errors or click **Install** to begin the installation.
 
-8. Scroll up to review the configuration settings and fix any errors or **Save** to complete the installation.
    The **Installed** screen displays with the **nextcloud** app in the **Deploying** state.
    It changes to **Running** when ready to use.
    Click **Web Portal** on the **Application Info** widget to open the Nextcloud web portal sign-in screen.
 
 {{< trueimage src="/images/SCALE/Apps/NextcloudSignInScreen.png" alt="Nextcloud Sign In Screen" id="Nextcloud Sign In Screen" >}}
 
 ## Troubleshooting Tips
+The Nextcloud apps with releases earlier than 2.0.4 are known to have issues. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.
+For more information on known issues, click [here](https://ixsystemsinc.slack.com/archives/C072YK2QZKM/p1715369467541109).
```

Comment:
Appears to be an incorrect link here. Maybe you meant to copy a link from slack and accidentally linked to Slack? Readers won't be able to access this.

Reply:
That is a bad link! I'll go get the Jira ticket link.

Reply:
Corrected the link.